components/region_cache_memory_engine/src/engine.rs

```diff
@@ -225,7 +226,18 @@ impl RangeCacheMemoryEngine {
         core.range_manager.new_range(range);
     }
 
-    pub fn evict_range(&mut self, range: &CacheRange) {
+    /// Load the range in the in-memory engine.
+    // This method only push the range in the `pending_range` where sometime
```

Comment:
```suggestion
    /// This method push the range to `pending_range` where
```

Reply:
May be mention `prepare_for_apply` here, which is where the snapshot loading actually happens.

Reply:
Done.